<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LibraryExplorer</class>
 <widget class="QWidget" name="LibraryExplorer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="library_explorer_layout">
     <item>
      <layout class="QHBoxLayout" name="library_search_layout">
       <item>
        <widget class="QLineEdit" name="library_search_input">
         <property name="minimumSize">
          <size>
           <width>294</width>
           <height>22</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Recherche dans la Bibliothèque</string>
         </property>
         <property name="maxLength">
          <number>255</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="library_search_button">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Rechercher dans la bibliothéque</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/icons.qrc">
           <normaloff>:/utility/search.png</normaloff>:/utility/search.png</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="library_explorer_displaymode">
         <property name="minimumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>33</width>
           <height>29</height>
          </size>
         </property>
         <property name="toolTip">
          <string>Mode d'affichage ...</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset resource="icons/icons.qrc">
           <normaloff>:/utility/setup.png</normaloff>:/utility/setup.png</iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTreeViewClickable" name="library_explorer_tree">
       <property name="minimumSize">
        <size>
         <width>376</width>
         <height>238</height>
        </size>
       </property>
       <property name="acceptDrops">
        <bool>true</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DropOnly</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QTreeViewClickable</class>
   <extends>QTreeView</extends>
   <header>qtreeviewclickable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons/icons.qrc"/>
 </resources>
 <connections/>
</ui>
